shoppingcart/server/main.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/pborman/uuid"
+	"github.com/temporalio/samples-go/shoppingcart"
+	enumspb "go.temporal.io/api/enums/v1"
+	"go.temporal.io/sdk/client"
+	"log"
+	"net/http"
+	"sort"
+)
+
+var (
+	cartState      = shoppingcart.CartState{Items: make(map[string]int)}
```

Comment:
I don't think cart state should be held in a global variable accessible in the client web app. The point of modeling a cart as a temporal workflow is that the storage, and point-of-truth, for the cart contents is the workflow itself. Using this global variable makes that less clear to the readers IMO. But I'm just proposing a small change: have `listHandler` take the cart state as an argument.